### PR TITLE
mention mysql replication

### PR DESCRIPTION
**Pull request by lance6716**
### What is changed, added or deleted? (Required)

Chinese part of https://github.com/pingcap/docs/issues/5137

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: mysql-compatibility.md [hunk 1]
```diff
@@ -13,6 +13,10 @@ aliases: ['/docs-cn/dev/mysql-compatibility/','/docs-cn/dev/reference/mysql-comp
     - 目前对这些功能的需求度不高，例如存储流程和函数。
     - 一些功能在分布式系统上的实现难度较大。
 
+- 除此以外，TiDB 不支持 MySQL 复制协议，但提供了专用工具用于与 MySQL 复制数据
+    - 从 MySQL 复制：[TiDB Data Migration (DM)](https://docs.pingcap.com/zh/tidb-data-migration/stable/overview) 是将 MySQL/MariaDB 数据迁移到 TiDB 的工具，可用于增量数据的复制。
+    - 向 MySQL 复制：[TiCDC](https://github.com/pingcap/ticdc) 是一款通过拉取 TiKV 变更日志实现的 TiDB 增量数据同步工具，可通过 [MySQL sink](/ticdc-overview.md#sink-支持) 将 TiDB 增量数据复制到 MySQL。
```
Comment (TomShawn) [inline mysql-compatibility.md:18]:
```suggestion
    - 向 MySQL 复制：[TiCDC](https://github.com/pingcap/ticdc) 是一款通过拉取 TiKV 变更日志实现的 TiDB 增量数据同步工具，可通过 [MySQL sink](/ticdc/ticdc-overview.md#sink-支持) 将 TiDB 增量数据复制到 MySQL。
```

**Review by yikeke — APPROVED** [review]

**Comment by TomShawn** [comment]
@lance6716 Could you invite a technical review? Thanks!

**Review by leoppro — APPROVED** [review]

File: mysql-compatibility.md [hunk 2]
```diff
@@ -13,6 +13,10 @@ aliases: ['/docs-cn/dev/mysql-compatibility/','/docs-cn/dev/reference/mysql-comp
     - 目前对这些功能的需求度不高，例如存储流程和函数。
     - 一些功能在分布式系统上的实现难度较大。
 
+- 除此以外，TiDB 不支持 MySQL 复制协议，但提供了专用工具用于与 MySQL 复制数据
+    - 从 MySQL 复制：[TiDB Data Migration (DM)](https://docs.pingcap.com/zh/tidb-data-migration/stable/overview) 是将 MySQL/MariaDB 数据迁移到 TiDB 的工具，可用于增量数据的复制。
+    - 向 MySQL 复制：[TiCDC](https://github.com/pingcap/ticdc) 是一款通过拉取 TiKV 变更日志实现的 TiDB 增量数据同步工具，可通过 [MySQL sink](/ticdc/ticdc-overview.md#sink-支持) 将 TiDB 增量数据复制到 MySQL。
```
Comment (overvenus) [inline mysql-compatibility.md:18]:
```suggestion
    - 向 MySQL 复制：[TiCDC](https://docs.pingcap.com/zh/tidb/stable/ticdc-overview) 是一款通过拉取 TiKV 变更日志实现的 TiDB 增量数据同步工具，可通过 [MySQL sink](/ticdc/ticdc-overview.md#sink-支持) 将 TiDB 增量数据复制到 MySQL。
```

Comment (lance6716) [reply mysql-compatibility.md:18]: maybe I copied it from another place. I'll check if we need update them tomorrow.

File: mysql-compatibility.md [hunk 3]
```diff
@@ -13,6 +13,10 @@ aliases: ['/docs-cn/dev/mysql-compatibility/','/docs-cn/dev/reference/mysql-comp
     - 目前对这些功能的需求度不高，例如存储流程和函数。
     - 一些功能在分布式系统上的实现难度较大。
 
+- 除此以外，TiDB 不支持 MySQL 复制协议，但提供了专用工具用于与 MySQL 复制数据
+    - 从 MySQL 复制：[TiDB Data Migration (DM)](https://docs.pingcap.com/zh/tidb-data-migration/stable/overview) 是将 MySQL/MariaDB 数据迁移到 TiDB 的工具，可用于增量数据的复制。
+    - 向 MySQL 复制：[TiCDC](https://docs.pingcap.com/zh/tidb/stable/ticdc-overview) 是一款通过拉取 TiKV 变更日志实现的 TiDB 增量数据同步工具，可通过 [MySQL sink](/ticdc/ticdc-overview.md#sink-支持) 将 TiDB 增量数据复制到 MySQL。
```
Comment (TomShawn) [inline mysql-compatibility.md:18]:
```suggestion
    - 向 MySQL 复制：[TiCDC](/ticdc/ticdc-overview.md) 是一款通过拉取 TiKV 变更日志实现的 TiDB 增量数据同步工具，可通过 [MySQL sink](/ticdc/ticdc-overview.md#sink-支持) 将 TiDB 增量数据复制到 MySQL。
```

**Comment by TomShawn** [comment]
/lgtm
@overvenus PTAL again

**Pull request merged**